src/components/FindWallet/WalletTable.tsx

```diff
@@ -0,0 +1,1098 @@
+            classNamePrefix="react-select"
+            options={[
+              {
+                label: "Choose to compare",
+                options: [...filteredFeatureDropdownItems],
+              },
+            ]}
+            onChange={(selectedOption) => {
+              setThirdFeatureSelect(selectedOption)
+              trackCustomEvent({
+                eventCategory: "WalletFeatureCompare",
+                eventAction: `Select WalletFeatureCompare`,
+                eventName: `${selectedOption.filterKey} selected`,
+              })
+            }}
+            defaultValue={thirdFeatureSelect}
+            isSearchable={false}
+          />
+        </th>
+      </WalletContentHeader>
+      {filteredWallets.map((wallet, idx) => {
+        const deviceLabels: Array<string> = []
+
+        wallet.ios && deviceLabels.push("iOS")
+        wallet.android && deviceLabels.push("Android")
+        wallet.linux && deviceLabels.push("Linux")
+        wallet.windows && deviceLabels.push("Windows")
+        wallet.macOS && deviceLabels.push("macOS")
+        wallet.chromium && deviceLabels.push("Chromium")
+        wallet.firefox && deviceLabels.push("Firefox")
+        wallet.hardware && deviceLabels.push("Hardware")
+
+        return (
+          <WalletContainer>
+            <Wallet>
+              <td>
+                <FlexInfo>
+                  <div>
+                    <Image
+                      image={getImage(data[wallet.image_name])!}
+                      objectFit="contain"
+                    />
+                  </div>
+                  <div>
+                    <p>{wallet.name}</p>
+                    <SecondaryText>{deviceLabels.join(" | ")}</SecondaryText>
+                    {deviceLabels.map((label) => (
+                      <SecondaryTextMobile>{label}</SecondaryTextMobile>
+                    ))}
+                    <SocialsContainer>
+                      <Socials>
+                        <Link
+                          to={wallet.url}
+                          hideArrow={true}
+                          customEventOptions={{
+                            eventCategory: "WalletExternalLinkList",
+                            eventAction: `Go to wallet`,
+                            eventName: `${wallet.name} ${idx}`,
+                            eventValue: filters,
+                          }}
+                        >
+                          <Icon name="webpage" size={"1.5rem"} color={true} />
+                        </Link>
+                        {wallet.twitter && (
+                          <Link
+                            to={wallet.twitter}
+                            hideArrow={true}
+                            customEventOptions={{
+                              eventCategory: "WalletExternalLinkList",
+                              eventAction: `Go to wallet`,
+                              eventName: `${wallet.name} ${idx}`,
```

Comment:
We want to track the position a wallet was in to know if users are going down the list to find a preferred wallet, or just picking the top position ones. Once we add the ranking feature this will likely improve a lot. 

cc @konopkja